std_msgs/Header header
string command_type
string command_data
int32 priority
float64 timestamp 